.github/ISSUE_TEMPLATE/bug-report-.md

```diff
@@ -7,6 +7,6 @@ assignees: ''
 
 ---
 
-- [ ] are you running newest version (version from docs) ?
+- [ ] what version of (W)(GL)Makie are you running?
```

Comment:
```suggestion
- [ ] what version of (W)(GL)(Cairo)Makie are you running?
```
but this could get confusing.  Maybe:
```suggestion
- [ ] what version of `Makie` and its backend (`(W/GL/Cairo)Makie` are you running?  (e.g. `]st -m Makie GLMakie`)
```

Reply:
Honestly just the Makie version is enough. The backends are released in synch with Makie and we use the Makie version for pretty much everything.

Reply:
Usually yes but not always - the backends can sometimes desync back to the AbstractPlotting days where there was no Makie compat...@kellertuer recently had a similar issue and posted on Slack about it, for example.

Reply:
Wouldn't that still give us either an ancient version or none?

Reply:
That's true...I guess we can always ask for the backend version in the issue!

Reply:
```suggestion
- [ ] what version of Makie are you running? (`]st -m Makie`)